sources/Valkey.Glide/Internals/Request.BitmapCommands.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+using Valkey.Glide.Commands.Options;
+
+using static Valkey.Glide.Internals.FFI;
+
+namespace Valkey.Glide.Internals;
+
+internal partial class Request
+{
+    public static Cmd<long, bool> GetBitAsync(ValkeyKey key, long offset)
+        => new(RequestType.GetBit, [key.ToGlideString(), offset.ToGlideString()], false, response => response != 0);
+
+    public static Cmd<long, bool> SetBitAsync(ValkeyKey key, long offset, bool value)
+        => new(RequestType.SetBit, [key.ToGlideString(), offset.ToGlideString(), (value ? 1 : 0).ToGlideString()], false, response => response != 0);
+
+    public static Cmd<long, long> BitCountAsync(ValkeyKey key, long start = 0, long end = -1, StringIndexType indexType = StringIndexType.Byte)
+    {
+        List<GlideString> args = [key.ToGlideString(), start.ToGlideString(), end.ToGlideString()];
+        if (indexType != StringIndexType.Byte)
+        {
+            args.Add(indexType.ToLiteral().ToGlideString());
+        }
+        return Simple<long>(RequestType.BitCount, [.. args]);
+    }
+
+    public static Cmd<long, long> BitPositionAsync(ValkeyKey key, bool bit, long start = 0, long end = -1, StringIndexType indexType = StringIndexType.Byte)
+    {
+        List<GlideString> args = [key.ToGlideString(), (bit ? 1 : 0).ToGlideString(), start.ToGlideString(), end.ToGlideString()];
```

Comment:
You've got this repeated in a few places. Is it worth adding a `ToGlideString` extension for `bool` so that we can directly do these conversions? I imagine that there are a few other places that could use that?


```suggestion
        List<GlideString> args = [key.ToGlideString(), bit.ToGlideString(), start.ToGlideString(), end.ToGlideString()];
```